io.github.craigrow.sparrow.yaml

```diff
@@ -0,0 +1,29 @@
+app-id: io.github.craigrow.sparrow
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: sparrow
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  - name: sparrow
+    buildsystem: simple
+    build-commands:
+      - cp -R sparrow/bin/ /app/sparrow
+      - chmod +x /app/sparrow/sparrow
+      - mkdir /app/bin/
+      - mkdir /app/lib/
+      - ln -s /app/sparrow/sparrow /app/bin/sparrow
+      - install -Dm644 sparrow/com.craigrow.sparrow.metainfo.xml /app/share/appdata/com.craigrow.sparrow.metainfo.appdata.xml
+      - install -Dm644 sparrow/icons/256x256/com.craigrow.sparrow.png /app/share/icons/hicolor/256x256/apps/com.craigrow.sparrow.png
+      - install -Dm644 sparrow/com.craigrow.sparrow.desktop /app/share/applications/com.craigrow.sparrow.desktop
+    sources:
+      - type: archive
+        url: "https://github.com/Strozer2002/sparrow/releases/download/v1.0.0/sparrow-wallet.tar.xz"
```

Comment:
This is x86_64 only binary and libs, won't work on aarch64.


Also why is it coming from a fork of flathub/flathub? Where is the source repo?

Reply:
I add my repo on github

Reply:
My app exporting from Windows and worked only on X86_64

Reply:
You need https://docs.flathub.org/docs/for-app-authors/maintenance#only-build-on-x86_64 then

Reply:
Sorry, its my first app on linux, i add json